deploy/canary/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:latest
```

Comment:
Moved canary -> beta for builds we make manually. The automated build will be the canary. The thought is that users can try a fixed version without us having to make a release. We can probably replace a lot of the "try vmarmol/cadvisor" with this.